website/siteConfig.js

```diff
@@ -74,7 +74,7 @@ const siteConfig = {
     'https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.0/clipboard.min.js',
     // Mathjax for rendering math content
     `${baseUrl}js/mathjax.js`,
-    'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-AMS_HTML',
+    'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-AMS_SVG',
```

Comment:
```suggestion
    'https://cdn.jsdelivr.net/npm/mathjax@2/MathJax.js?config=TeX-AMS_SVG',
```

Reply:
> An alternative fix could be to upgrade our mathjax version as later versions may not be as reliant on local STIX fonts but the 3.0 update was a complete rewrite and appears to not be a straightforward upgrade.

Let's not bother spending time on the 3.0 upgrade at this point. But how about doing this which [should give us the latest 2.x MathJax version](https://www.mathjax.org/#gettingstarted)?

Reply:
That makes sense to me! Tested this change to verify everything continues to work as expected